### ISSUE TITLE
See instance path of the gpu

### ISSUE THREAD

**Issue by vpnxen** [VirtualDisplay/Virtual-Display-Driver#222]
Greetings,

i want to know is there anyway to find out which GPU the Virtual adapter is attached to ( the instance path of it 
something like this:
\\?\PCI#VEN_10DE&DEV_2684&SUBSYS_88EF1043&REV_A1#4&38ec4dcd&0&0019#{064092b3-625e-43bf-9eb5-dc845897dd59}\GPUPARAV

i have same gpu and i put the name in the xml , but because it is have same name i dont know which one it connected to.
is there anyway to figure it out?

i test gwmi win32_VideoController , but it show it as a new video controller 

but i saw something like this the log that is for runtime RenderAdapterLuid: 90668-0 and not static , can you help me please?

**Comment by bud3699** [comment]
Uh, in the latest beta does not display luid however in the next release I added luid logging support to also log the luid of the gpu it's using. For now you may be able to get away with dxdiag

**Comment by vpnxen** [comment]
you kn

> Uh, in the latest beta does not display luid however in the next release I added luid logging support to also log the luid of the gpu it's using. For now you may be able to get away with dxdiag
> 
> You may be able to compare in properties in device manager. But I'm not at home to check
> 
> It will be the first gpu which displays in device manager
> 
> So if you have two 2060s it will be the first one which shows in device manager

thank you sir, 
you know i have 7x 4090 on a system rig , and they are run in head less , without any monitor, and a create vm on them , but sunshine do not find any monitor.
do you know how i can handle this case , i for test connect a monitor to one of 4090's , it fixed , but i want handle it via virtual display.

if you have any suggestion i will be appreciate it

**Comment by vpnxen** [comment]
yes , but the issue is there is no monitor connected to my host, and all of them work in head less , but once i connect a monitor sunshine in vm works fine , i want some how fool the GPU to think it is a monitor attached to it.

**Comment by bud3699** [comment]
I think it genuinely might be a power management issue on gpus where if no physical display is connected then the gpu disconnects as I cannot reproduce it, mine works completely fine with no displays plugged in. The virtual display adapter stays active

**Comment by zjoasan** [comment]
Sorry for late reply, have you tried a HDMI-dummy instead of that "real monitor"?
